spec/etl/master/master_processor_spec.rb

```diff
@@ -18,10 +18,19 @@
     allow(Master::MetricsProcessor).to receive(:process)
   end
 
-  it 'does not process if already processed for date' do
-    create(:dimensions_date, date: Date.yesterday)
+  describe 'runs only once per day' do
+    it 'raises `DuplicateDateError` if there are already metrics for the day' do
+      ddate = create(:dimensions_date, date: Date.yesterday)
```

Comment:
Not sure but this looks like a typo with `ddate`? Otherwise GTG.

Reply:
It was a shorter way for `dimensions_date`, but I have updated it as it is obviously not clear enough :)